src/main/java/com/cse/ngsa/app/utils/ParameterValidator.java

```diff
@@ -1,5 +1,6 @@
 package com.cse.ngsa.app.utils;
 
+import com.cse.ngsa.app.controllers.BenchmarkController;
```

Comment:
This creates a circular reference and dependency between ParameterValidator and BenchmarkController.  My guess is that you're attempting to reference from ParameterValidator back to a property of BenchmarkController.   Let's remove this import and fix up the dependency.